routes/cards.js

```diff
@@ -0,0 +1,86 @@
+const express = require('express');
+const knex = require('../knex');
+const { camelizeKeys, decamelizeKeys } = require('humps');
+
+const router = express.Router();
+
+router.post('/cards', (req, res, next) => {
+  const { question, answer, resourceUrl } = req.body;
+
+  knex('cards')
+  .insert(decamelizeKeys({ question, answer, resourceUrl }), '*')
+  .then((newCard) => {
+    res.send(camelizeKeys(newCard));
+  })
+  .catch(err => next(err));
+});
+
+router.get('/cards/:id', (req, res, next) => {
+  const { id } = req.params;
+
+  knex('cards')
+  .where('id', id)
+  .first()
+  .then((row) => {
```

Comment:
I would replace this `row` and the subsequent ones with the resource name. In this case `card`